watertap/examples/flowsheets/electrodialysis/electrodialysis_1stack.py

```diff
@@ -0,0 +1,326 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+from pyomo.environ import (
+    ConcreteModel,
+    value,
+    Constraint,
+    Objective,
+    TransformationFactory,
+    assert_optimal_termination,
+)
+from pyomo.network import Arc
+
+from idaes.core import FlowsheetBlock, UnitModelCostingBlock
+from idaes.core.solvers import get_solver
+from idaes.core.util.initialization import propagate_state
+from idaes.core.util.model_statistics import degrees_of_freedom, report_statistics
+from idaes.models.unit_models import Feed, Product, Separator
+from pandas import DataFrame
+import idaes.core.util.scaling as iscale
+import idaes.logger as idaeslogger
+from watertap.unit_models.electrodialysis_1D import Electrodialysis1D
+from watertap.costing.watertap_costing_package import (
+    WaterTAPCosting,
+)
+from watertap.property_models.ion_DSPMDE_prop_pack import DSPMDEParameterBlock
+
+__author__ = "Xiangyu Bi"
+
+
+def main():
```

Comment:
If you call this main function in the test file (i.e. test_main_function or something), all of this code will be covered and the display functions would be covered

Reply:
Added a simple test to run main().